### ISSUE TITLE
instrument stacks

### ISSUE THREAD

**Issue by brendangregg** [draios/sysdig#43]
A stack field class could be included, with stack.kernel and stack.user for retrieving each stack back trace. This would allow quick identification of the reason events occurred: code path ancestry.

A minor detail: it doesn't matter if stacks are returned multi-line (easier to read) or single-line (easier to post process). Eg, for single line, frames can be separated by ';'. 

(Stack trace data can also be easily fed into Flame Graphs...)

**Comment by lovasko** [comment]
Exactly, I was just exploring sysdig and was wondering if I could use the Flame graphs by @brendangregg :+1: 

Is there any plan for this?

**Comment by dsummersl** [comment]
+1!

**Comment by acornejo** [comment]
+1

**Comment by unlikely-leo** [comment]
Any movement on this issue? We'd be very interested in using `sysdig` to profile our staging clusters. @ashmere @mwitkow
